### DISCUSSION TITLE
Module customization

### DISCUSSION THREAD

**Discussion by ArniSamano** [YetiForceCompany/YetiForceCRM#16489] [How to]
Hi All

1. I have a need to customize certain module, so it not showing fields name (for instance). What would be the best approach to make changes to individual modules without touching core files? 
2. How to remove floating "Full Screen" from text fields? It messing a bit with quick edit pencil
 
Tnx.

**Comment by apcloic** [comment]
@ArniSamano
I'm not sure I understand what you want to do with fields name but you can use loader/autoloader mechanism which allow you to customize your system without touching core files. 
https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/item/loader-and-autoloader-of-files
#13831
#12070
Regards,

**Comment by s2b-git** [comment]
How to override styles of certain module with autoloader?

**Reply by apcloic** [reply]
@s2b-git afaik, autoloader does not allow you to override styles for now, it works for files in /modules, /layouts and /languages.

**Reply by mariuszkrzaczkowski** [reply]
there is an additional style file loading mechanism, styles are generated from scss files so you cannot overwrite one
